.github/scripts/check_migrations.py

```diff
@@ -0,0 +1,54 @@
+import subprocess
+import sys
+
+
+def get_diff():
+    subprocess.run(
+        ["git", "fetch", "origin"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL
+    )
+    output = [
+        f"./{file_path}"
+        for file_path in subprocess.check_output(
+            ["git", "diff", "--name-only", "origin/develop"]
```

Comment:
`origin/develop` what will happen if the operations are moved to new branch names?

Reply:
What do you mean by operations? If we change `develop` and `main`? I think that might be over-engineering if we have some sort of variable we can pass in there - it would still need to be set somewhere 🙂

Reply:
Yes indeed, hopefully we don't have any over-engineering 🙂 
I am thinking,  `origin/develop`  `origin/main` should stand out from the bulk of code - maybe module level var, or better a constant file, as they are important points - to view and consider should any changes to them are to be made.